### PR TITLE
Automatically detect and alert when v1 and v2 file changes drift

### PR DESCRIPTION
**Pull request by zephraph**
This adds a script to help ensure we're considering what files need to be changed in v1 and v2.

### CODE REVIEW

File: scripts/schema-drift.ts [hunk 1]
```diff
@@ -0,0 +1,378 @@
+const { promises: fs, createReadStream } = require("fs")
+const path = require("path")
+const { createHash } = require("crypto")
+const { spawn } = require("child_process")
+
+const schemaV1Path = path.join(process.cwd(), "src/schema/v1")
+const schemaV2Path = path.join(process.cwd(), "src/schema/v2")
```
Comment (alloy) [inline scripts/schema-drift.ts]: Better to make relative to `__dirname`

File: scripts/schema-drift.ts [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+    delta.on("close", code => {
+      if (code !== 0) {
+        reject("Failed to get branch drift")
+      } else {
+        const commitChanges = output.match(/(\d+)\s+(\d+)/)
+        if (!commitChanges) {
+          reject("Something was wrong with the branch drift output")
+        }
+
+        let [, commitsBehind, commitsAhead] = Array.from(commitChanges!).map(
```
Comment (alloy) [inline scripts/schema-drift.ts]: Ooh, what does that initial comma do? Does it make you select the remainder from the tail of the list?

File: scripts/schema-drift.ts (hunk 1 above)
Comment (alloy) [reply scripts/schema-drift.ts]: Oh I didn’t realize yarn did that, that’s cool! 👌

File: scripts/schema-drift.ts (hunk 2 above)
Comment (alloy) [reply scripts/schema-drift.ts]: Ah ok, it just skips one. Can you have consecutive commas?

**Comment by mzikherman** [comment]
Will do!

**Pull request merged**